kpi/serializers/v2/asset_permission_assignment.py

```diff
@@ -27,24 +28,22 @@
 )
 from kpi.utils.urls import absolute_resolve
 
-
 ASSIGN_OWNER_ERROR_MESSAGE = "Owner's permissions cannot be assigned explicitly"
 
 
 class AssetPermissionAssignmentSerializer(serializers.ModelSerializer):
-
```

Comment:
💔 . Your linter is going to be crazy on all my code. I do add an extra line in purpose.

Reply:
It happens several times in this PR. I would keep the extra line at the beginning of classes (and methods).